sections_text/README.md

```diff
@@ -4487,3 +4487,28 @@ Here we use some of the classes of the [card](https://materializecss.com/cards.h
 - Finally, test on your browser
 
 Finally, we finished our goals for this app but this is just the beginning of our journey because will be your job to expand and refactor the app. Good look and thanks for reading this.
+
+## Git workflow
+
+On this repository I going to implement a `git workflow` that will allow us to accomplish our goals on a consitent and productive manner using git in the most effective way that we can. I defined the following specifications
```

Comment:
Conventional commits is a great tool also, https://www.conventionalcommits.org/en/v1.0.0/
I believe could be useful

Reply:
Yes; the plan is to add that after adding the linter here we just specify the documentation of the git workflow that we are going to follow